docs/development/developer-setup.md

```diff
@@ -71,6 +71,35 @@ docker compose up
 bin/setup
 ```
 
+## Using sanitised production data
+
+You may occasionally want to use data as close to production data ase possible, for instance during an incident or when preparing for a large migration.
+
+### 1. Downloading the sanitised data
+
+- Login to azure with your `digitalauth.education.gov.uk` address and request a PIM (speak to infra or a fellow developer for instructions on how to do this)
```

Comment:
I think people without a `digitalauth.education.gov.uk` email address can't get a PIM for production. That was my understanding when I set one up. I can't really do anything in azure when I'm logged in with my `education.gov.uk`. 

I don't think we want to get overly specific -- this could change and we don't want to replicate other instructions here. The important part is 'speak to infra or a fellow developer for instructions on how to do this'

Reply:
I use `education.gov.uk`,  I don't even have a `digitalauth` email, only contractors do. Permanent members of staff may be confused and think that they need to set up a `digitalauth` email.

I totally agree about not getting overly specific and replicating other instructions. Maybe we could change it to something like: 

> You will need to request a PIM for production data (speak to infra or a fellow developer for instructions on how to do this)

Either way, definitely not a blocker.